ioctl/cmd/action/actiondeploy.go

```diff
@@ -19,7 +19,7 @@ import (
 var actionDeployCmd = &cobra.Command{
 	Use:   "deploy [AMOUNT_IOTX] [-s SIGNER] -b BYTE_CODE [-l GAS_LIMIT] [-p GAS_PRICE] [-P PASSWORD] [-y]",
 	Short: "Deploy smart contract on IoTeX blockchain",
-	Args:  cobra.MaximumNArgs(0),
+	Args:  cobra.MaximumNArgs(1),
```

Comment:
if args = 0, how come the deploy can work now?

Reply:
no parameter, amount_iotx will be 0